odoo/addons/base/models/res_config.py

```diff
@@ -472,7 +474,7 @@ def _get_classified_fields(self):
             elif name.startswith('module_'):
                 if field.type not in ('boolean', 'selection'):
                     raise Exception("Field %s must have type 'boolean' or 'selection'" % field)
-                module = IrModule.sudo().search([('name', '=', name[7:])], limit=1)
+                module = IrModule._get(name[7:])
```

Comment:
One day we'll be able to use `str.removeprefix` (though I guess it has its own oddities)